src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
+    public int getUserScore(String[] records, String email) throws UserNotFoundException {
+
+        for (int i = 0; i < records.length; i++) {
+            if (email.equals(records[i].substring(0,records[i].indexOf(':')))) {
```

Comment:
let's make this solution simpler:
- firstly we can split `record` by ":" and save the result to the array.
- then we can compare the first value from the array with the email
- if comparing returns true - you should return the second value from the array.

Reply:
done